kbase-extension/static/kbase/js/widgets/function_output/kbaseGenericSetViewer.js

```diff
@@ -48,26 +48,32 @@ define([
             this._super(options);
 
             const $self = this;
-            if (options._obj_info) {
-                $self.obj_info = options._obj_info;
-                $self.obj_ref =
-                    $self.obj_info.ws_id + '/' + $self.obj_info.id + '/' + $self.obj_info.version;
-                $self.link_ref =
-                    $self.obj_info.ws_id + '/' + $self.obj_info.name + '/' + $self.obj_info.version;
-                //$self.update_overview_info_from_nar_info($self.obj_info);
-            } else {
-                $self.obj_ref = $self.options.wsId + '/' + $self.options.objId;
-                $self.link_ref = $self.obj_ref;
-                $self.set_overview.name = $self.options.objId;
-                //$self.set_overview.link_ref = $self.link_ref;
+            if (!options._obj_info) {
+                console.warn('no _obj_info: input was')
+                console.warn(options)
+                return
+                // if (options.obj_ref && options.wsName) {
```

Comment:
This can probably be removed, as it's basically there below.

Reply:
This was an accidental commit from something else I was troubleshooting. Not sure why I didn't see it when I was reviewing the PR! I'll remove it.